CHANGELOG.md

```diff
@@ -1,5 +1,34 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2023-01-22
+
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/).
+
+### Announcements
+
+* Starting with Kubernetes 1.26:
+  * HostProcess Containers will be GA
+  * Some AKS labels will be deprecated. Update your AKS labels to the recommended substitutions. See more information on label deprecations and how to update your labels in the [Use labels in an AKS cluster](https://docs.microsoft.com/azure/aks/use-labels) documentation.
+* AKS began pod security policy deprecation on 2022-11-01 API. The [pod security policy](https://learn.microsoft.com/azure/aks/use-pod-security-policies) will be removed completely on 2023-06-01 API with AKS 1.25 version or higher. You can migrate pod security policy to [pod security admission controller](https://learn.microsoft.com/azure/aks/use-psa) before the deprecation deadline.
+* Azure Policy will be updated to [GateKeeper 3.11](https://github.com/open-policy-agent/gatekeeper/releases/tag/v3.11.0) on Feb 20th for AKS 1.24 and up.
```

Comment:
can we move this to component updates section?

Reply:
@miwithro we have released aksmanagedAutoUPgradeSchedule that is specifically targeted for Auto Upgrade Schedules and offers better controls, flexibility like quarterly , biweekly, bimonthly etc. Read more [Use Planned Maintenance for your Azure Kubernetes Service (AKS) cluster (preview) - Azure Kubernetes Service | Microsoft Learn](https://learn.microsoft.com/en-us/azure/aks/planned-maintenance)we have released aksmanagedAutoUPgradeSchedule that is specifically targeted for Auto Upgrade Schedules and offers better controls, flexibility like quarterly , biweekly, bimonthly etc. Read more [Use Planned Maintenance for your Azure Kubernetes Service (AKS) cluster (preview) - Azure Kubernetes Service | Microsoft Learn](https://learn.microsoft.com/en-us/azure/aks/planned-maintenance)

Reply:
This is pre-announcement, meaning technically this shouldn't have happened yet, and it would be part of of components when it is. In this case though it seems 3.11 will be for all, so probably just having in components when it's done is enough.

Reply:
add Auto Upgrade Feature @kaarthis.  Modified the text for the AKS patch versions @qpetraroia.  Everyone else please review and Approve.